src/App.tsx

```diff
@@ -15,36 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SORT {
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+  DEFAULT = '',
+}
+
+function sort(goods: string[], sortField: SORT, isReversed: boolean): string[] {
```

Comment:
Function names should start with a verb to make it clear what the function does. The function `sort` could have a more descriptive name like `sortGoods`.